site/content/docs/5.1/components/navbar.md

```diff
@@ -306,7 +307,11 @@ Mix and match with other components and utilities as needed.
 
 ## Color schemes
 
-Theming the navbar has never been easier thanks to the combination of theming classes and `background-color` utilities. Choose from `.navbar-light` for use with light background colors, or `.navbar-dark` for dark background colors. Then, customize with `.bg-*` utilities.
+{{< callout warning >}}
+**New in v5.2.0:** Navbar theming is now powered by CSS variables and `.navbar-light` has been deprecated. CSS variables are applied to `.navbar`, defaulting to the "light" appearance, and can be overridden with `.navbar-dark`.
+{{< /callout >}}
+
+Navbar themes are easier than ever thanks to Boosted's combination of Sass and CSS variables. The default is our "light navbar" for use with light background colors, but you can also apply `.navbar-dark` for dark background colors. Then, customize with `.bg-*` utilities.
```

Comment:
I understand. It is a bit troubling maybe since all the examples in this page with Boosted context are dark navbars. But that still explain that you could override it if needed and also that's the reason we have everywhere `.navbar-dark`. It doesn't shock me too much either. Have you a proposal maybe?

Reply:
As it is explained a bit above, I'm not sure if the last sentence is worth. However, I'd say : 

Navbar themes are easier than ever thanks to Boosted's combination of Sass and CSS variables. The default is our "light navbar" that shouldn't be used in Orange websites, but you can also apply `.navbar-dark` for dark background colors.

Reply:
Tried something with 271a605f7efc5f66eaf717609f1e8d681c5e5b3b. Is it clearer?